### PR TITLE
Liang- Homework1

### PR DESCRIPTION
**Pull request by feng-liang**
Below I provide a list of steps I follow to finish this assignment:
1. Fork the repository into my github account.
2. Create a new project in R Studio, open the repo using version control-Git
3. Make changes in R studio
4. Commit the change to Git, and push it to github
5. Create pull request to update the homework to turn in assignment.

To make changes to the repo in R Studio are fairly easy and straightforward with the Mastering Markdown guideline. 

I think my biggest challenges is that I did not realize I can change the name when I create the pull request. I try to change the name README.md to Liang-README HW1 locally and that created a tons of unnecessary files for me. It greatly interfere with my commit/push process because there are many conflicts in versions. In retro-respect, I am still not sure if I should've changed the file name.

### CODE REVIEW

**Comment by mmdeng1** [comment]
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | x |   |  
Coding strategy | x |   |  
Presentation | x |   |  
Achievement, creativity | x |   |  
Ease of access | x |   |  

Hi, Liang. Though your work is relatively short, I found all those required elements--good job. I especially like the bold text you highlighted in your introduction. It sheds light on your suggestion which personally, I found very helpful too. But I do believe you should incorporate a picture of your interest to your assignment. Also, I think you could make your brief bio a bit more creative if you put your random list along with your general introduction organically--for instance, you can build a table about your backgrounds, your hobbies, things like that. Overall, I think you did a good job for this assignment, but I definitely believe if you can spend some extra time on this, you can make your project more attractive.

**Comment by GabrielKahn** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |   x        |              |            |

Remarks:
Hi Liang, it is very nice to meet you. Your code is simple and effective, overall giving a very clean template that presents the project clearly.

Your coding strategy hits your marks well, achieving a readable page to learn more about you.

Your presentation is fine. It is very short and to the point, which is good! But you also have more room to play around with R-Markdown.

For achievement and creativity, I thought listing your favorite cafe's around campus was very useful and creative. I will have to try more of them out. More lists of such use would boost your grade. Also you can use this opportunity to provide a picture of your favorite café. Either of these approaches would have given you an excellent mark.

For ease of access, I had no problem accessing your file, so full marks for you!

Thank you for your submission, I enjoyed it.

**Review by kangkanak — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |   X        |              |            |
| **Presentation**            |    X       |              |            |
| **Achievement, creativity** |        |   X           |            |
| **Ease of access**          |     X      |              |            |

Remarks: Overall the presentation is nice! I like the use of italics/bold and the inclusion of a link that was helpful in completing the assignment. The inclusion of a picture would have been nice though.

File: Liang-README HW1.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## General Introduction about Myself
+My name is Feng Liang and I am a MAPSS student in sociology. My thesis is very qualitative oriented and almost does not involve any manipulation of numeric data at all. It is very refreshing for me to sit in this class and to think about new ways of interacting with data. Here's link to the [Markdown guide](https://guides.github.com/features/mastering-markdown/#examples) that is **enormously** helpful for my assignment.
+
+### Random list about my favorite cafe on campus
```
Comment (bensoltoff) [inline Liang-README HW1.md:4]: You don't need to increment the numbers yourself. That will happen when the document is rendered. For instance,

```
1. Ground of Being
1. Starbuck on 55TH
1. Pret
1. Dollop
1. Ex-Lib
```

becomes

1. Ground of Being
1. Starbuck on 55TH
1. Pret
1. Dollop
1. Ex-Lib

File: Liang-README HW1.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## General Introduction about Myself
+My name is Feng Liang and I am a MAPSS student in sociology. My thesis is very qualitative oriented and almost does not involve any manipulation of numeric data at all. It is very refreshing for me to sit in this class and to think about new ways of interacting with data. Here's link to the [Markdown guide](https://guides.github.com/features/mastering-markdown/#examples) that is **enormously** helpful for my assignment.
```
Comment (bensoltoff) [inline Liang-README HW1.md:2]: Best practice is to leave a line break between different types of content. For example, leave a blank line between a header and a block of text.

```
## General Introduction about Myself
My name is Feng Liang and I am a MAPSS student in sociology. My thesis is very qualitative oriented and almost does not involve any manipulation of numeric data at all. It is very refreshing for me to sit in this class and to think about new ways of interacting with data. Here's link to the [Markdown guide](https://guides.github.com/features/mastering-markdown/#examples) that is **enormously** helpful for my assignment.
```

It makes it easier to read the original `.md` file, plus also prevents rendering errors later trying to view the document as an HTML or PDF file.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |         x     |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |     x      |              |            |

**Pull request closed without merging**